lib/muchos/config/base.py

```diff
@@ -244,13 +276,15 @@ def has_service(self, service):
                 return True
         return False
 
+    # test method, might want to make private or just move to test module
```

Comment:
yes it should be pretty straight forward, especially since python has no notion of variable hiding or protection, test code can access and/or modify any internal variables just fine.  Making private in python land is adding underscore prefix(es) and is more a convention than a rule.